cli/internal/test/cli.go

```diff
@@ -15,23 +16,29 @@ type FakeCli struct {
 	command.DockerCli
 	client     client.APIClient
 	configfile *configfile.ConfigFile
-	out        io.Writer
+	out        command.OutputStreamer
 	err        io.Writer
-	in         io.ReadCloser
+	in         command.InputStreamer
+	store      credentials.Store
 }
 
-// NewFakeCli returns a Cli backed by the fakeCli
+// NewFakeCli returns a Cli backed by the fakeCli handling the creation of the OutputStreamer using a Writer
 func NewFakeCli(client client.APIClient, out io.Writer) *FakeCli {
+	return NewFakeCliBase(client, command.NewOutStream(out))
+}
+
+// NewFakeCliBase returns a Cli backed by the fakeCli
```

Comment:
If we don't introduce `OutputStreamer` (and I really think we shouldn't), this is not required.